.circleci/workflows.yml

```diff
@@ -531,10 +582,19 @@ workflows:
 commands:
   ferrocene-job-build:
     description: Collection of steps for build jobs
+    parameters:
+      os:
+        type: string
+        default: unix # Or windows
```

Comment:
Bearing in mind we'll support Mac we should make this `linux`, `windows` or (eventually) `darwin`